utils/lib/rr_graph/points.py

```diff
@@ -12,10 +12,10 @@
 from collections import defaultdict
 from collections import namedtuple
 
-from ..asserts import assert_eq
-from ..asserts import assert_type
+from lib.asserts import assert_eq
```

Comment:
Why did you change these imports?

Reply:
Invoking doctest explodes without it.

Reply:
Go into the symbiflow-arch-defs/utils directory and run `python -m lib.rr_graph.points`?